### ISSUE TITLE
The code have many problem

### ISSUE THREAD

**Issue by xiexukang** [MenghaoGuo/-EANet#5]

**Comment by xiexukang** [comment]
Some basic pytorch code errors 
nn.softmax????

**Comment by JackEasson** [comment]
It uses jittor, not pytorch ...

**Comment by xiexukang** [comment]
sorry,my mistakes
